packages/sources/deutsche-boerse/src/endpoint/lwba.ts

```diff
@@ -44,13 +52,6 @@ interface LwbaLatestPriceResponse {
 
 export type BaseEndpointTypes = {
   Parameters: typeof inputParameters.definition
-  Response: LwbaLatestPriceResponse
+  Response: LwbaLatestPriceResponse | LwbaMetadataResponse
   Settings: typeof config.settings
 }
```

Comment:
Can we do separate BaseEndpointTypes for each endpoint instead of having variable response types? 
Would be a bit cleaner to read and for Streams schema to parse.